lib/include/public/CommonFields.h

```diff
@@ -51,6 +51,8 @@
 #define COMMONFIELDS_EVENT_INITID                            "EventInfo.InitId"
 #define COMMONFIELDS_EVENT_SEQ                               "EventInfo.Sequence"
 #define COMMONFIELDS_EVENT_PRIVTAGS                          "EventInfo.PrivTags"
+#define COMMONFIELDS_EVENT_PRIVDATACATEGORY                  "EventInfo.PrivDataCategory"
+#define COMMONFIELDS_EVENT_PRIVPRODUCT                       "EventInfo.PrivProduct"
```

Comment:
Any plans to add the Java convenience API to set these fields here - https://github.com/microsoft/cpp_client_telemetry/blob/c838e74db34da8c7266770c8d2a282a3f916102f/lib/android_build/maesdk/src/main/java/com/microsoft/applications/events/EventProperties.java (similar to `setPrivacyTags`). 

@maxgolov Do you know if we need to modify [JSONFormatter](https://github.com/microsoft/cpp_client_telemetry/commit/409c139c7579a7cf426a85810fc3bb80c4de4c52#diff-ba9e29e66788b4539c1a2bf02576e46663f530f72e95b44a79a3fb7ec5e7b7e8) to handle these fields? I can't see where we use this JSON serializer ?

Reply:
I am trying to address the needs of native consumers. Maybe a tracking item can be added.